lib/NameAddrHeader.js

```diff
@@ -32,7 +32,7 @@ module.exports = class NameAddrHeader
     // Initialize parameters.
     this._uri = uri;
     this._parameters = {};
-    this._display_name = display_name;
+    this.display_name = display_name;
```

Comment:
No need to remove the underscore. The attribute has its own getter and setter in the class.

Reply:
I removed the underscore so that `this._display_name` would get assigned via the display_name setter function and `display_name`s of integer 0 would get converted to `"0"`.

This allowed for the removal of duplicated logic handling integer 0 in the toString() method.